### DISCUSSION TITLE
23.05 正式版已经发布，好像官网可以更新了

### DISCUSSION THREAD

**Discussion by CallMeR** [immortalwrt/immortalwrt#1095] [General]
> Upcoming Stable Release
ImmortalWrt 23.05 will be the successor of the 21.02 series. It is currently in the release candidate phase and will replace the current stable release once finalized.

https://downloads.immortalwrt.org/

**Comment by microka** [comment]
https://t.me/ctcgfw_openwrt_discuss/1053738

> 最近 OpenWrt 发布了 23.05.0 正式版，但与 23.05.0-rc4 没有本质上的区别，没有重要的更新。
> 由于每次发版将占用约 110G 磁盘空间，ImmortalWrt 暂时不会跟随发版。
